stage1/enter_kvm/enter_kvm.go

```diff
@@ -63,9 +63,16 @@ func getPodDefaultIP(workDir string) (string, error) {
 	}
 	// kvm flavored container must have at first position default vm<->host network
 	if len(nets) == 0 {
-		return "", fmt.Errorf("Pod has no configured networks")
+		return "", fmt.Errorf("pod has no configured networks")
 	}
-	return nets[0].IP.String(), nil
+
+	for _, net := range nets {
+		if net.NetName == "default" || net.NetName == "default-restricted" {
+			return net.IP.String(), nil
+		}
+	}
+
+	return "", fmt.Errorf("pod has no default network!")
```

Comment:
This would be the case if `--net=none` is chosen, or are there other cases? Do we warn the user at this point? 

Making sure: does the kvm flavor load the *default-restricted* network if the user passes in other network names?